codec-http2/src/test/java/io/netty/handler/codec/http2/internal/hpack/DecoderTest.java

```diff
@@ -33,16 +33,20 @@
 
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.Unpooled;
+import io.netty.buffer.UnpooledByteBufAllocator;
```

Comment:
Definitely. Interestingly Checkstyle does not appear to be setup to detect this.